apps/smartling/frontend/vite.config.ts

```diff
@@ -12,13 +12,6 @@ export default defineConfig(() => ({
     // the parent project that combines both the frontend and the hosted app action backend needs to be able to override
     // the default location to its own build path
     outDir: process.env.BUILD_PATH || './build',
-    rollupOptions: {
```

Comment:
turns out this didn't really fix the caching issue so I removed it here, still need to determine why the cache is temporarily stale on staging until you clear cache/hard reload but not a blocking issue for the work in this PR.